icicle/appUtils/ntt/ntt.cu

```diff
@@ -311,14 +333,14 @@ void ntt_inplace_batch_template(
         <<<num_blocks, num_threads, 0, stream>>>(d_inout, n, d_twiddles, n, total_tasks, s, false);
     }
 
-    if (is_coset) batch_vector_mult(coset, d_inout, n, batch_size, stream);
+    if (is_coset) batchVectorMult<E, S><<<num_blocks, num_threads, 0, stream>>>(coset, d_inout, n, batch_size);
 
     num_threads = max(min(n / 2, MAX_NUM_THREADS), 1);
     num_blocks = (n * batch_size + num_threads - 1) / num_threads;
     template_normalize_kernel<E, S>
       <<<num_blocks, num_threads, 0, stream>>>(d_inout, n * batch_size, S::inv_log_size(logn));
   } else {
-    if (is_coset) batch_vector_mult(coset, d_inout, n, batch_size, stream);
+    if (is_coset) batchVectorMult<E, S><<<num_blocks, num_threads, 0, stream>>>(coset, d_inout, n, batch_size);
```

Comment:
the sneak case is a better idea imho - is now sort of an implicit defacto standard for our code?

Reply:
➕ Lets be consistent across the codebase, all snake case (our current) or all camel case (Google C++ style, CUDA Runtime style)

Reply:
Yea I wanted to make functions snake-case on CUDA side, as per google C++ style guide. Is everyone OK with that @jeremyfelder @ImmanuelSegol @ChickenLover

Reply:
I'm fine either way but 

>Yea I wanted to make functions snake-case on CUDA side, as per google C++ style guide

I think google c++ style guide says function names are UpperCamelCase: https://google.github.io/styleguide/cppguide.html#Function_Names

If we want to have Google c++ style as a base we can add this to the clang-format configuration file: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#basedonstyle